glslangValidator symlink cannot be created in a distro package build environment

Version: 12.3.1

```
CMake Error: failed to create symbolic link 'glslangValidator': Read-only file system
```

Basically, cmake tries to symlink into /usr/bin/glslangValidator -> glslang, but it completely ignores $DESTDIR. In a distro packaging environment DESTDIR will be set somewhere (in my case, to /buildstream-install/), and the rest of the system (including /usr) will be mounted read-only.

--- a/StandAlone/CMakeLists.txt
+++ b/StandAlone/CMakeLists.txt
@@ -121,7 +121,7 @@ if(ENABLE_GLSLANG_INSTALL)
     # Create the same symlink at install time
     install(CODE "execute_process( \
                       COMMAND ${CMAKE_COMMAND} -E ${link_method} $<TARGET_FILE_NAME:glslang-standalone> ${legacy_glslang_name} \
-                      WORKING_DIRECTORY ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR})")
+                      WORKING_DIRECTORY \$ENV{DESTDIR}/${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR})")
 
     if(ENABLE_SPVREMAPPER)
         install(TARGETS spirv-remap EXPORT glslang-targets)

